src/plugins/LoadImbalanceThroughSleepPlugin.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "PluginBase.h"
+
+/**
+ * Plugin that produces a load imbalace by sleeping.
+ *
+ * Current modes:
+ * - "varyingSteps" -- introduces need for overlapping globals.
+ *     - all even ranks: sleep at afterForces
+ *     - all uneven ranks: sleep at endStep
+ */
+class LoadImbalanceThroughSleepPlugin : public PluginBase {
+public:
+	/**
+	 * \code{.xml}
+	 * <plugin name="LoadImbalanceThroughSleepPlugin">
+	 *   <varyingSteps>True/False</varyingSteps> <!-- Default: True -->

```

Comment:
so what happens if I set this to false? The plugin is active and does nothing?
I would either remove this option change it so that it accepts a string-name for the mode if you think it is worthwhile to implement different modes in the same plugin. But probably one should implement other plugins for this.

Reply:
as written above:
this is one of the supported modes.
currently there is, sadly only one mode implemented ^^

-> meant for extensibility ;)